cli/main.rs

```diff
@@ -349,6 +349,7 @@ async fn run_subcommand(flags: Arc<Flags>) -> Result<i32, AnyError> {
   handle.await?
 }
 
+#[cfg(feature = "panic-hook")]
```

Comment:
setup_panic_hook should happen either way, its just the small part around `deno_panic::trace` that needs to be cfg'd